datalad/interface/tests/test_save.py

```diff
@@ -81,7 +81,9 @@ def test_save(path):
     with open(opj(subds.path, "some_file.tst"), "w") as f:
         f.write("something")
     subds.add('.')
-    ok_clean_git(subds.path, annex=isinstance(ds.repo, AnnexRepo))
+    ok_clean_git(subds.path, annex=isinstance(subds.repo, AnnexRepo))
```

Comment:
Furthermore, what you suggest is a matter of line 78: `subds = ds.create('subds')`, which has no parameter `no_annex=True`. If this is to be tested, then it has to be done in a unittest for `create`, not here.